src/modules/form.js

```diff
@@ -0,0 +1,95 @@
+
+const radioButtons = [
+  ['kelvin', '01'], ['celsius', '02'], ['fahrenheit', '03'],
+];
+
+const units = {
+  '01': '',
+  '02': '&units=metric',
+  '03': '&units=imperial',
+};
+
+function createRadioBtn(radioContainer, radioButtons, i) {
+  const radioBox = creator(radioContainer, 'div', 'append');
+
+  const radio = creator(radioBox, 'input', 'append');
+  radio.setAttribute('type', 'radio');
+  radio.setAttribute('value', `${radioButtons[i][1]}`);
+  radio.setAttribute('id', `${radioButtons[i][0]}`);
+  radio.setAttribute('class', 'weather-input radio-input');
+  radio.setAttribute('name', 'units');
+  const label = creator(radioBox, 'label', 'append');
+  label.setAttribute('for', `${radioButtons[i][0]}`);
+  label.innerHTML = `${radioButtons[i][0].charAt(0)
+    .toUpperCase()
+    .concat(radioButtons[i][0]
+      .slice(1))}`;
+  if (i === 1) radio.checked = true;
+}
+
+function addCBToSearchBtn(button, main, units) {
+  button.addEventListener('click', async (e) => {
+    e.preventDefault();
+    const inputs = document.getElementsByClassName('weather-input');
+    const values = getInputsValues(inputs);
+    if (values) {
+      const cityName = modifyInput(values[0], units[values[1]]);
+      const data = await getWeather(cityName);
+      if (data) {
+        const allData = filterData(data);
+        setBGImage(iconList, allData[6]);
+        setValues(symbols, ...allData);
+      } else {
+        const modalContent = createModal(main);
+        modalContent.innerHTML = 'The city provided could not be found.';
+        document.getElementById('error-msg').style.display = 'block';
+      }
+    } else {
+      const modalContent = createModal(main);
+      modalContent.innerHTML = 'Fill in required field.';
+      document.getElementById('error-msg').style.display = 'block';
+    }
+  });
+}
+
+function formSearch(parent, units) {
+  const form = creator(parent, 'form', 'append');
+
+  const searchContainer = creator(form, 'div', 'append');
+  searchContainer.setAttribute('id', 'search-container');
+
+  const input = creator(searchContainer, 'input', 'append');
+  input.setAttribute('name', 'search');
+  input.setAttribute('type', 'text');
+  input.setAttribute('placeholder', 'Search...');
+  input.setAttribute('class', 'weather-input');
+  input.required = true;
+
+  const button = creator(searchContainer, 'button', 'append');
+  button.setAttribute('type', 'submit');
+  button.setAttribute('id', 'search-btn');
+  addCBToSearchBtn(button, parent, units);
+
+  const icon = creator(button, 'span', 'append');
+  icon.setAttribute('class', 'material-icons');
+  icon.innerHTML = 'search';
+
+  const priorityTitle = creator(form, 'p', 'append');
+  priorityTitle.innerHTML = 'Select Unit Format:';
+  const radioContainer = creator(form, 'div', 'append');
+  radioContainer.setAttribute('class', 'radio-container');
+
+  for (let i = 0; i < radioButtons.length; i += 1) {
+    createRadioBtn(radioContainer, radioButtons, i);
+  }
```

Comment:
**You are using `radio button` kindly checkout the screenshot from your web page below;**
![image](https://user-images.githubusercontent.com/31897434/92688852-f8ba5a00-f335-11ea-983b-b22b7aef7ba2.png)

**You should use a toggle control to determine if the result will be shown in Celsius or Fahrenheit** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))